main/webapp/modules/core/scripts/index/open-project-ui.js

```diff
@@ -167,7 +236,7 @@ Refine.OpenProjectUI.prototype._renderProjects = function(data) {
                   if (!found) {
                       project.userMetadata.push({
                           name: data.customMetadataColumns[m].name,
-                          dispay: data.customMetadataColumns[m].display,
+                          display: data.customMetadataColumns[m].display,
```

Comment:
@mayralgr I clicked on the **About** blue link next to one of my files in the Project listing, and then added a custom Title, then Apply, then went back to do a Search and see if it would filter on that Project file row, but it seemed to just hang the entire browser and process?
However, I am not sure if this is your fault.

@wetneb Do you think this is probably some leftover issue around the Metadata or Serialization of it and we can skip the testing for Searchability for the customMetadataColumns until we know more? or maybe it's something simple she can change here to make it work?